test/onnx/test_fx_type_promotion.py

```diff
@@ -0,0 +1,27 @@
+# Owner(s): ["module: onnx"]
+
+import pytorch_test_common
+from torch.onnx._internal.fx.passes import type_promotion
+from torch.testing._internal.common_utils import run_tests, TestCase
+
+
+class TestGeneratedTypePromotionRuleSet(TestCase):
+    @pytorch_test_common.skip_in_ci(
+        "Reduce noise in CI. "
+        "The test serves as a tool to validate if the generated rule set is current. "
```

Comment:
When should we run this?

Reply:
When there is type related issues and someone follows `Note add type promotion rule` to check if generated set is current.